src/txn/txn_rollback_to_stable.c

```diff
@@ -42,9 +42,10 @@ __rollback_abort_newer_update(
             first_upd = upd->next;
 
             __wt_verbose(session, WT_VERB_RTS,
-              "%s: Rollback to stable update aborted with durable timestamp: %s and stable "
-              "timestamp: %s",
-              S2BT(session)->dhandle->name, __wt_timestamp_to_string(upd->durable_ts, ts_string[0]),
+              "%s: Rollback to stable update aborted with txnid: %" PRIu64
+              " durable timestamp: %s and stable timestamp: %s",
+              S2BT(session)->dhandle->name, upd->txnid,
+              __wt_timestamp_to_string(upd->durable_ts, ts_string[0]),
```

Comment:
You don't need to prefix the object name in the message, that's done automatically by the message handling routines.

Reply:
Fixed.